app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

```diff
@@ -0,0 +1,1722 @@
+    api: {
+      param: "candidate.english_language_qualifications",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["unknown"],
+      what: "Further research required"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    notes: ["Does having an English language GCSE (or equivalent) mean we need to ask this question? Does an English GCSE indicate fluency?"]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Other languages spoken",
+  items: {
+    question: "If you are bilingual or very familiar with languages other than English, you can tell us about them here",
+    required: "No",
+    api: {
+      param: "candidate.other_languages",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
+    },
+    why: [{
+      who: ["candidate"],
+      what: "may want to demonstrate language skills, especially if looking to teach a language subject or teach in school with children whose main language isn’t English"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["Assesed at interview"],
+    notes: ["SCITTs/SDs we've recently spoken to haven’t expressed a need for this, but it’s something candidates sometimes like to mention"]
+  }
+}) }}
+
+## Contact details
+
+A candidate can update these details at any point during the application process.
+
+{{ appProtocol({
+  title: "Email address",
+  items: {
+    question: "Enter your email address",
+    required: "Yes",
+    api: {
+      param: "contact_details.email",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#contactdetails-object"
+    },
+    why: [{
+      who: ["service"],
+      what: "requires this to send candidates a ‘magic link’ to sign back in, and to send updates about their application"
+    }, {
+      who: ["provider"],
+      what: "requires this to contact candidates to arrange interviews and send communications relevant to their application"
+    }, {
+      who: ["hesa"],
+      what: "requires this as the “DfE requires this to communicate with newly qualified teachers”"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: [
+      "Address format is validated",
+      "Service requires a verified email address to sign in"
+    ]
+  }
+}) }}
+
+{{ appProtocol({
+  title: "Phone number",
+  items: {
+    question: "Phone number",
+    required: "Yes",
+    api: {
+      param: "contact_details.phone_number",
+      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#contactdetails-object"
+    },
+    why: [{
+      who: ["provider"],
+      what: "may use this to contact a candidate"
+    }],
+    what: ["Appears in provider-facing interface and retrievable via API"],
+    accuracy: ["❓"],
```

Comment:
We validate format.

But we also allow international numbers without their country code (this has happened)

Reply:
Updated note.